src/status_im2/contexts/shell/cards/view.cljs

```diff
@@ -25,7 +25,9 @@
                               :size           :small
                               :override-theme :dark
                               :label          (i18n/label :t/kicked)}]
-       (:count :permission) [:<>]) ;; Add components for these cases
+       (:count :permission) [:<>]  ;; Add components for these cases
+
+       nil)
```

Comment:
Adding default nil values for switcher `case` uses, so that app doesn't crash if gets unmatch key.